pkgs/games/fheroes2/default.nix

```
@@ -0,0 +1,44 @@
+{ stdenv, lib, fetchFromGitHub,
+  gettext, libpng, SDL2, SDL2_image, SDL2_mixer, SDL2_ttf, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "fheroes2";
+  version = "0.9.4";
+
+  src = fetchFromGitHub {
+    owner = "ihhub";
+    repo = "fheroes2";
+    rev = version;
+    sha256 = "sha256-z+88tVsf4uyMFzNfZDKXo0cYqBCYn1ehX+A+e+aIfSg=";
+  };
+  nativeBuildInputs = [ gettext libpng SDL2 SDL2_image SDL2_mixer SDL2_ttf zlib ];
```

```suggestion
  };

  buildInputs = [ gettext libpng SDL2 SDL2_image SDL2_mixer SDL2_ttf zlib ];
```

or are any binaries called from those packages?

---

No, I misunderstood the difference between those two.